install/kubernetes/istio-gateway.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: istio-gateway
+spec:
+  selector:
+    istio: ingressgateway # use istio default controller
+  servers:
+  - port:
```

Comment:
For master - we won't expose http ports, only grpc.

Reply:
yeah, 15007 is not required here

Reply:
Unless we use internal load balancer - we can't expose plain text insecure stuff over internet.

With MTLS - it is fine to expose internal Istio components in a normal gateway.